cf/rds.yaml

```diff
@@ -0,0 +1,123 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Set up RDS Aurora cluster [managed by github.com/hms-dbmi-cellenics/iac]
+
+Parameters:
+  Environment:
+    Type: String
+    Default: development
+    AllowedValues:
+      - development
+      - staging
+      - production
+    Description: The environment for which the cluster needs to be created.
+
+# --------------------------------------- Constants ---------------------------------------
+  DBPort:
+    Type: String
+    Default: "5432"
+
+  DBEngineVersion:
+    Type: Number
+    Default: 13.4
+
+  DBSecretName:
+    Description: The name of the secret used to store master user and password.
+    Type: String
+    Default: aurora-staging
+
+Resources:
+  DBCluster:
+    Type: AWS::RDS::DBCluster
+    DeletionPolicy: Snapshot
+    UpdateReplacePolicy: Snapshot
+    Properties:
+      Engine: aurora-postgresql
+      EngineMode: provisioned
+      EngineVersion: !Ref DBEngineVersion
+      DBSubnetGroupName: !Ref DBSubnetGroup
+      DBClusterIdentifier: !Join [ "-", ["aurora-cluster", !Ref Environment] ]
+      DatabaseName: "aurora_db"
+      Port: !Ref DBPort
+      # Can't be deleted before first setting this to false, should be true when we are done with the basic setup of db
+      DeletionProtection: false
```

Comment:
delete protection means that the rds database itself can't be removed?

Reply:
yes, it is kind of a safety switch, you can't delete a database with `DeletionProtection: true`

Reply:
You can delete the instances, but the cluster will still be around (so all the stored data will be around as well).